app/helpers/a11y_helper.rb

```diff
@@ -15,4 +19,18 @@ def graphical_image(source, options = {})
   def image(source, alt, options = {})
     image_tag source, { alt: alt }.merge(options)
   end
+
+  private
+
+  def font_awesome_class(style)
+    style_class = case style
+                  when :normal
```

Comment:
This should be `legacy` I think. `fa` has been replaced by `far` (regular), `fal` (light), and `fas` (solid). We use a CSS stub to deal with this for now, as its a bit hard to make the update as previously the weights were defined in CSS.

Reply:
In here https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4, `fa` == `fas`, but I agree that we should have a legacy style still just to make sure that we haven't broken anything.